common/wforce-webserver.hh

```diff
@@ -146,6 +146,8 @@ public:
       drogon::app().disableSession();
       drogon::app().disableSigtermHandling();
       drogon::app().setLogLevel(trantor::Logger::kWarn);
+      // We will never allow uploads, but drogon wants to create a bunch of temp files in uploadPath/tmp/xx
+      drogon::app().setUploadPath("/tmp/wforce");
```

Comment:
Maybe use `std::filesystem::temp_directory_path` ?

Reply:
Ok, not such a good idea as it turn out, not for C++ reasons but for toolchain reasons:
```
/usr/bin/ld: wforce-lua.o: in function `WforceWebserver::init()':
3779
/wforce/wforce/../common/wforce-webserver.hh:151: undefined reference to `std::filesystem::temp_directory_path[abi:cxx11]()'
```

Support for std::filesystem is a something of a toolchain nightmare, and one I don't want to get into for this patch. So I'm going to revert this change for now.